tests/test_benchmark.py

```diff
@@ -455,20 +454,17 @@ def test_hif2a():
     benchmark_hif2a(verbose=True, num_batches=2, steps_per_batch=100)
 
 
-def get_nonbonded_pot_params(bps: List[BoundPotential]) -> Optional[Tuple[BoundPotential[Nonbonded], NDArray]]:
-    for bp in bps:
-        if isinstance(bp.potential, SummedPotential):
-            for pot, params in zip(bp.potential.potentials, bp.potential.params_init):
-                if isinstance(pot, Nonbonded):
-                    return pot, params
-    return None, None
+def get_nonbonded_pot_params(bps):
+    try:
+        pot, params = next((bp.potential, bp.params) for bp in bps if isinstance(bp.potential, Nonbonded))
+        return pot, params
```

Comment:
Could you add a quick comment about this change? (Is it no longer nested?)

Reply:
With this PR, the `Nonbonded` potential is no longer nested inside of a `SummedPotential`, so we don't need to search inside of `SummedPotential`s. Can add a quick docstring if it would be helpful.

Reply:
Actually, was your question more about what this function does, or why the change was needed? Hopefully the above comment addresses why it was needed, and the simplification in https://github.com/proteneer/timemachine/pull/1087/commits/49e5755c9c13ffc046bae81059d20d8239d4d0e3 should make the code pretty self-explanatory

Reply:
Thanks for clarifying -- makes sense, and good additional simplification!